src/lando/main/scm/git.py

```diff
@@ -460,9 +460,11 @@ def clean_repo(
         # explicitly given an empty string.
         if attributes_override is not None:
             # $GIT_DIR/info/attributes has the highest precedence.
-            with open(f"{self._git_dir}/info/attributes", "w+") as fp:
-                if not "\n".join(fp.readlines()).startswith(attributes_override):
+            with open(f"{self._git_dir}/info/attributes", "a+") as fp:
```

Comment:
Not sure this needs to be `a+`. Maybe this should be `r+`. Or this can be split into two `open` statements (`r` and `w`) and then you don't need to truncate or seek altogether.